### ISSUE TITLE
aria-expanded is not announced in Chrome

### ISSUE THREAD

**Issue by nousrnm** [nvaccess/nvda#11274]
https://drive.google.com/file/d/1ZfpmAJz_-swwqdlwUYGipCYS7dSK74n2/view?usp=sharing

### Steps to reproduce:
Open the sub navigation: https://youtu.be/ixtKsrX_-pc
Close the sub navigation by pressing Enter when focus is on MEN: https://youtu.be/i_VyP_3Tc1Y
Close the sub navigation by pressing Tab + Shift when focus is on MEN: https://youtu.be/CixqtdL5QM4

### Actual behavior:
Nothing is announced.

### Expected behavior:
NVDA should announce "Expanded" or "Collapsed" based on the new value.

When you open the sub navigation: https://youtu.be/R_iXH78cC9Y
When you close the sub navigation by pressing Enter when focus is on MEN: https://youtu.be/sd9NSX-jcsc
When you close the sub navigation by pressing Tab + Shift when focus is on MEN: https://youtu.be/nxFXkGH03iw

#### NVDA installed/portable/running from source:
Installed.

#### NVDA version:
2020.1

#### Windows version:
10 Home, Version 2004.

#### Name and version of other software in use when reproducing the issue:
Google Chrome, Version 83.0.4103.106 (Official Build) (64-bit)

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.

#### If addons are disabled, is your problem still occuring?
I don't have any addons.

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No.

If I remove these parts of the code, NVDA works as expected, so maybe it's a Chrome problem...
https://drive.google.com/file/d/16FxHluJM7l1iFFPXZOnG1gTM0yZ1GvuM/view?usp=sharing

**Comment by Adriani90** [comment]
This is because the button has an aria has popup on it. This is a duplicate of #9096 and might be solved either in NVDA or in Chrome itself. I am closing as duplicate.

**Comment by nousrnm** [comment]
If you download the file that contains the code:
https://drive.google.com/file/d/1ZfpmAJz_-swwqdlwUYGipCYS7dSK74n2/view?usp=sharing

You will see that the button doesn't have an aria-haspopup attribute.

**Comment by khushveer** [comment]
@Adriani90 The issue here is not that it does not announce the state change, just that it announces that when i navigate away from the element and not while the action is being performed.
Please see the attached video for reference.  Also, can we please reopen this issue?


Browser Versions:

Microsoft Edge version 81.0.416.58 (Official build) (64-bit)
Windows Version: windows 10
Google Chrome version 81.0.4044.113 (Official build) (64-bit)

Code Snippet:
`<div>`
`<div>`
`<span class="caption">Filters</span>`
`<span class="example">(e.g., Activity ext:cs)</span>`
`</div>`
`<ul role="listbox" class="list detailed" aria-label="Search filters list">`
`<li role="option" class="item" tabindex="-1">`
`<div class="text">ext:</div>`
`<div class="hint">With file extension</div>`
`</li>`
`<li role="option" class="item" tabindex="-1">`
`<div class="text">def:</div>`
`<div class="hint">Definition</div>`
`</li>`
`<li class="item expander" aria-expanded="false" role="button" tabindex="-1" aria-label="Show more">`
`<span aria-hidden="true" class="expander-icon flex-noshrink fabric-icon ms-Icon--ChevronDownMed small"></span>`
`<span>Show more</span>`
`</li>`
`</ul>`
`</div>`

[BugVideo_3.zip](https://github.com/nvaccess/nvda/files/4809587/BugVideo_3.zip)

**Comment by Adriani90** [comment]
Ok, thanks for the code snippet. This is more clear now, it seems related to the aria hidden attribute.
cc: @feerrenrut, @ObjectInSpace,
I am reopening this issue.

**Comment by nousrnm** [comment]
> it seems related to the aria hidden attribute.
I tried to say this 3 days ago, and I think I did a pretty good job [explaining it](https://github.com/nvaccess/nvda/issues/11277#issuecomment-646421364)...

> I am reopening this issue.
But there is another topic about the same issue... https://github.com/nvaccess/nvda/issues/11277

**Comment by Adriani90** [comment]
@nousrnm ok thanks for the clarification. I am closing this again, having in mind issue #11277. For the future it would be ideal to have minimal test cases or code snoppets when create such new issues. Otherwise people can get confused and investigation is really hard or even impossible.
Thanks for reporting and let's continue the discussion on #11277.
